docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -4006,11 +4006,11 @@ JSON Parameters:
     - **Resources** – Resource requirements which apply to each individual container created as part
       of the service.
         - **Limits** – Define resources limits.
-            - **CPU** – CPU limit
-            - **Memory** – Memory limit
+            - **NanoCPUs** – CPU limit in units of `1/1e9` CPU shares.
```

Comment:
Thinking if we can come with a better description of the unit; 10<sup>−9</sup> (one billionth)?

/cc @aluzzardi @sfsmithcha any suggestions?